Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM golang:1.22.1-alpine
+
+COPY . .
+
+
+RUN go get .
+RUN go build
+
+EXPOSE 8080
+
+CMD ["./airport-app-backend"]
```

Comment:
Please research and use multi-stage builds. This current one will result in a very heavy image